src/main/java/seedu/address/model/ingredient/IngredientName.java

```diff
@@ -38,6 +38,14 @@ public static boolean isValidIngredientName(String ingredientName) {
         return Arrays.asList(INGREDIENTS).contains(ingredientName);
     }
 
+    /**
+     * Returns the name of the ingredient in string format.
+     * @return the name of the ingredient in string format.
+     */
+    public String getIngredientNameString() {
+        return ingredientName;
+    }
+
     @Override
     public String toString() {
         return ingredientName;
```

Comment:
Difference between these two functions?

Reply:
They are the same, thanks for noting. I will delete the function above.